application/frontend/config/main.php

```diff
@@ -21,14 +28,32 @@
             'enableSession' => false,
             'loginUrl' => null,
         ],
-        'session' => [
-            'cookieParams' => [// http://us2.php.net/manual/en/function.session-set-cookie-params.php
-                'lifetime' => $sessionLifetime,
-                'path' => '/',
-                'httponly' => true,
-                'secure' => $frontCookieSecure,
+        'sessionCache' => [
+            'class' => 'yii\caching\MemCache',
+            'servers' => [
+                [
+                    'host' => $memcacheHost1,
+                    'port' => $memcachePort1,
+                    'weight' => $memcacheWeight1,
+                ],
+                [
+                    'host' => $memcacheHost2,
+                    'port' => $memcachePort2,
+                    'weight' => $memcacheWeight2,
+                ],
             ],
         ],
+        'session' => [
+            'class' => 'yii\web\CacheSession',
+            'cache' => 'sessionCache',
+//            'cookieParams' => [// http://us2.php.net/manual/en/function.session-set-cookie-params.php
```

Comment:
This is still needed right? We need to write a session cookie to the browser for us to be able to find their session in memcache.